savant/deepstream/buffer_processor.py

```diff
@@ -606,7 +606,9 @@ def prepare_element_output(self, element: PipelineElement, buffer: Gst.Buffer):
                         # for object/complex models output - `bbox_tensor` and
                         # `selected_bboxes` - indices of selected bboxes and meta
                         # for attribute/complex models output - `values`
-                        bbox_tensor, selected_bboxes, values = None, None, None
+                        bbox_tensor: Optional[np.array] = None
+                        selected_bboxes: Optional[list] = None
```

Comment:
```suggestion
                        selected_bboxes: Optional[List] = None
```
Since we're still using python 3.8, it's better to use typing generics

Reply:
done